project-koku/koku
# [COST-4313] - Fix azure subscription_name issue

## Jira Ticket

[COST-4313](https://issues.redhat.com/browse/COST-4313)

## Description

This change will correct the missing subscription_name entry from the OCP on Azure api endpoints.

## Testing

1. Checkout Branch
2. Restart Koku
3. Load OCP on Azure data
4. Hit this endpoint `api/cost-management/v1/reports/openshift/infrastructures/azure/costs/?group_by[subscription_guid]=*`
5. See we now show subscription_name correctly

## Notes

Requires the following migration: https://github.com/project-koku/koku/pull/4934
...
